README.md

```diff
@@ -1892,6 +1892,48 @@ Started tailing... Press Ctrl+C to stop.
 ```
 Here is a screencapture showing the two commands above executed in the REPL. 
 ![tailing deferred](https://raw.githubusercontent.com/geoffhendrey/jsonataplay/main/taildefer.gif)
+## $rateLimit function
+Rate limiting allows to ensure than no than one function call is executed withing some time. For exameple,
+we want to ensure that a function calling external APIs does not overload it.
+```json
+>.init -f example/rateLimit.yaml
+{
+  "acc": [],
+  "appendAcc": "${ function($v){$set('/acc/-', $v)} ~> $rateLimit(100)}",
+  "counter": "${   function(){($set('/count', $$.count+1); $$.count)}    }",
+  "count": 0,
+  "rapidCaller": "${$setInterval(counter~>appendAcc, 10)}",
+  "stop": "${ count=100?($clearInterval($$.rapidCaller);'done'):'not done'  }",
+  "accCount": "${ $count(acc) }"
+}
+```
+Below output demonstrates, that `rateLimit` function calls to to set `acc` to once in no less than 100ms, which will 
+result in only 10 counts added o the `acc` array, the first one, the last one, and 10 in between. 
+```json ["data.accCounter = 12"]
+.init -f example/rateLimit.yaml --tail "/ until accCount=12"
+Started tailing... Press Ctrl+C to stop.
+{
+  "acc": [
```

Comment:
It will not surprise me if this fails intermittently in CI since you are relying on timing using a non-realtime OS. For example you can see the counts are not always multiples of 10. We see "28, 38, 48...65". Variance will be even higher in CI. JS makes no guarantee of the period of setInterval. It is just a best effort depending on what else the CPU is doing.